### ISSUE TITLE
Database creation error

### ISSUE THREAD

**Issue by DL6ER** [yubiuser/webchanges-docker#3]
I tried running this container from `:development` but it always fails to even start with my existing `urlwatch` configuration:

relevant `docker compose logs` lines:
```
webchanges  | Sat Dec  2 11:15:00 CET 2023
webchanges  | Running webchanges
webchanges  | sqlite3.OperationalError: unable to open database file
webchanges  | [15] Failed to execute script 'webchanges' due to unhandled exception!
```

**Comment by yubiuser** [comment]
I'll know what to do ..

**Comment by DL6ER** [comment]
Turns out: The directory `./data` needs to be writable by user with UID 1000

**Comment by DL6ER** [comment]
No, actually, I am going to repurpose/reopen this old issue here.

I did just notice that `webchanges-docker` started to send me the same update every minute. The issue - again - seems to have been that manual testing has changed the database ownership to `root:root` which caused `webchanges` to not be able to update the database. But: `docker compose logs` didn't mention it and there was also no other hint at this.

Is this expected to happen when the database is owned by `root`?
If so, would you agree that we'd need some sort of check here? I was only able to resolve it reasonably quick as I remembered this (one year old) issue and - after reading my comment above - decided to try this.

**Comment by yubiuser** [comment]
I think it is, as we have the crontab running as `webchanges`

https://github.com/yubiuser/webchanges-docker/blob/0e6a749443c17bbfc5ce7220aed625d681eea424/run.sh#L6-L7

So on a container start we set the permissions

https://github.com/yubiuser/webchanges-docker/blob/0e6a749443c17bbfc5ce7220aed625d681eea424/Dockerfile#L73-L75
